clicache/multiple-cache-integration-test/cache.xml

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<client-cache
+    xmlns="http://schema.pivotal.io/gemfire/gfcpp-cache"
```

Comment:
Need to be using the geode schema here.

Reply:
sounds like a ticket is needed to update schema used for all tests...